chsdi/templates/historicalmaps/viewer.mako

```diff
@@ -0,0 +1,295 @@
+<%namespace name="zeitreise_map" file="../zeitreihen_map.mako"/>
+
+<%
+  from pyramid.url import route_url
+  import urllib2
+  import json
+
+  c = context
+  request = c.get('request')
+  baseUrl = '//' + c.get('baseUrl')
+  layerBodId = c.get('layer')
+  featureId = c.get('bildnummer')
+  displayLink = True
+  lang = c.get('lang') if c.get('lang') is not None else 'de'
+  title = request.translate(layerBodId)
+  pageTitle = _(c.get('title')) + ': ' + featureId
+  title += ': ' + pageTitle
+  loaderUrl = h.make_agnostic(route_url('ga_api', request))
+  featureUrl = "https:%s/%s/%s/%s/%s" %( h.make_agnostic(route_url('topics',request)) ,'api','MapServer','ch.swisstopo.zeitreihen',featureId+"_"+c.get('release_year'))
+
+def feature_from_identify(featureUrl):
+    result = None
+    feature = None
+    response = None
+    try:
+        request = urllib2.Request(featureUrl)
+        response = urllib2.urlopen(featureUrl)
```

Comment:
It may be better to create a subrequest see: http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/subrequest.html
Alternatively, which is even better (avoid json dumps/loads) you should use the get_feature directly.